docs/DeveloperGuide.adoc

```diff
@@ -800,15 +800,59 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |create patient profile |register its information
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |find a patient by name |locate details of patient without having to go through the entire list
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |add appointments |schedule meeting with owners
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |find owner's contact number |
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* * *` |user |search for an animal's medical history |know next step of treatment
+
+|`* * *` |user |list out appointments for next day |know what to prepare
+
+|`* * *` |user |classify types of appointments |organize the appointments better
+
+|`* * *` |user |cancel an appointment |make space in schedule for other things
+
+|`* * *` |user |see future appointments |know how to plan my works
+
+|`* * *` |user |list out appointments for today |prepare for the work
+
+|`* * *` |user |list out patients of a specific species |contact them for blood donation
+
+|`* * *` |user |log in with passwords |protect my patients' information
```

Comment:
Changing to "protect my patients' and their respective owners' information" will be more specific and appropriate, because information of pets are not considered very crucial in the government's POV; it's the owners' information that falls under PDPA.